inc/Engine/Common/Ajax/AjaxHandler.php

```diff
@@ -5,19 +5,19 @@
 class AjaxHandler {
 
 	/**
-	 * Validate the referer before going futher.
-	 *
-	 * @param array $args Arguments to configure the validation.
+	 * Validate the referer before going further.
 	 *
+	 * @param string $action Action to validate.
+	 * @param string $capacities User capacity to verify.
 	 * @return bool
 	 */
-	public function validate_referer( array $args ) {
+	public function validate_referer( string $action = '', string $capacities = '' ) {
```

Comment:
the parameters are not optionals, so I don't think it's necessary to give them a default empty value. That will avoid a couple of conditionals in the method too